DH_Engine/Classes/DHATGun.uc

```diff
@@ -399,7 +397,6 @@ simulated function ClientEnterRotation()
     RotationProjector.bGradient = true;
     RotationProjector.SetDrawScale((2.5 * CollisionRadius)/RotationProjector.ProjTexture.MaterialUSize());
```

Comment:
Make sure there is whitespace between the `/` operator (and all operators, in fact).